configdemo/src/main/resources/application.yaml

```
@@ -6,7 +6,17 @@ spring:
+#  id: 122
+  version: ${VERSION}
+#  version: 2.2.2
+  name: ${NAME}
+#  name: "Default-Build"
+
+
+#  This is how we set environment variable from intellij IDE
+#  BUILD_ID=444444;BUILD_NAME=NAME_TEST;BUILD_VERSION=1.8.0
+
+# This is how we get dynamic value with ${} sign
+# ID=444444; NAME=NAME_TEST; VERSION=1.8.0
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)